src/prommis/solvent_extraction/tests/test_sx_model_dynamic.py

```diff
@@ -0,0 +1,261 @@
+from pyomo.environ import (
```

Comment:
Need the copyright header.

Reply:
What do you mean by copyright header?

Reply:
There is a standard header that needs to go at the top of every file that states the copyright for the code. You can look at some of the other examples to see it.

Reply:
I have added

Reply:
I don't see the copyright header here.

Reply:
I have added it.